nixpkgs/default.nix

```diff
@@ -1 +1,7 @@
-import ((import <nixpkgs> {}).fetchFromGitHub (builtins.fromJSON (builtins.readFile ./github.json)))
+# DO NOT HAND-EDIT THIS FILE
+import ((import <nixpkgs> {}).fetchFromGitHub (
+  let json = builtins.fromJSON (builtins.readFile ./github.json);
+  in { inherit (json) owner repo rev sha256;
+       private = json.private or false;
+     }
+))
```

Comment:
Let's restore this file to how it was before, for now.

Reply:
Thats just whatever `ob thunk` did.  Fixed!